library-management/src/app/components/renew-book/renew-book.component.html

```diff
@@ -0,0 +1,40 @@
+<app-user-functions></app-user-functions>
```

Comment:
app-user-functions> avoid using this in every function by using app.component.html